BC-NOV-20-Proyectos/alerta-covid-web-team02
# Iss25 modeling email styles

# Description
 - We gave styles to the emails that users who register on our platform and receive notifications through email will receive
 
 # Testing
- Clone our repo
- Start the server with rails s
- Generate a mail for you
- Go to http://localhost:3000/to see the welcome page

  ## Screenshots 
![Screenshot_2021-04-13-18-33-26-875_com google android gm](https://user-images.githubusercontent.com/42391583/115303808-b22da900-a129-11eb-869c-16b25482f972.png)
![Screenshot_2021-04-13-18-33-26-875_com google android gm](https://user-images.githubusercontent.com/42391583/115303828-bbb71100-a129-11eb-8046-bb0f6c069b03.png)
